R/build-home-index.R

```diff
@@ -25,7 +25,8 @@ build_home_index <- function(pkg = ".", quiet = TRUE) {
     dst_path,
     tweak_homepage_html,
     strip_header = strip_header,
-    sidebar = !isFALSE(pkg$meta$home$sidebar)
+    sidebar = !isFALSE(pkg$meta$home$sidebar),
+    bs_version = get_bs_version(pkg)
```

Comment:
I want to avoid using conditions on `bs_version` as much as possible, eventually.

Reply:
But it's also simpler for e.g. `build_news()`. :thinking: